upflux-client/src/components/ContactUs.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'; 
+import '../features/ContactUs/contactUs.css';
+import phoneIcon from '../assets/images/phone.png'; 
+import emailIcon from '../assets/images/email.png'; 
+
+const ContactUs = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    message: '',
+  });
+
+  const handleChange = (e: { target: { name: any; value: any; }; }) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({
+      ...prevData,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = (e: { preventDefault: () => void; }) => {
+    e.preventDefault();
+    console.log('Form submitted:', formData);
+    setFormData({ name: '', email: '', message: '' });
```

Comment:
please use typescript .. create interfaces or types for example `interface contactform {name:string; email:string; message: string}`

and then later you will have in ur methods as well strings rather than saying "any"

Reply:
Thanks for the feedback. I have changed it to match what you said.